Description: Makes vectorsctrip retain quality scores.
 This patch or an equivalent solution has been applied Upstream.
 https://sourceforge.net/tracker/index.php?func=detail&aid=2886368&group_id=93650&atid=605034
Origin: http://lists.open-bio.org/pipermail/emboss/2009-September/003735.html
--- emboss-6.1.0.orig/ajax/ajseq.c
+++ emboss-6.1.0/ajax/ajseq.c
@@ -782,6 +782,12 @@
     if (seq->Fttable)
 	pthis->Fttable = ajFeattableCopy(seq->Fttable);
 
+    if (seq->Accuracy)
+    {
+        AJCNEW0(pthis->Accuracy,seq->Seq->Len);
+        memmove(pthis->Accuracy,seq->Accuracy,seq->Seq->Len*sizeof(float));
+    }
+
     return pthis;
 }
 
